packages/contracts/deploy-config/sepolia.json

```diff
@@ -15,8 +15,7 @@
   "validatorPoolTrustedValidator": "0xc2da150ecfaa2e275577203bb177dd4de4a2536e",
   "validatorPoolMinBondAmount": "0x16345785d8a0000",
   "validatorPoolMaxUnbond": 10,
-  "validatorPoolNonPenaltyPeriod": 20,
-  "validatorPoolPenaltyPeriod": 40,
+  "validatorPoolRoundDuration": 60,
```

Comment:
If l2BlockTime is **2** and l2OutputOracleSubmissionInterval is **60**, shouldn't validatorPoolRoundDuration be **30**?

Reply:
```
ROUND_DURATION = (L2_BLOCK_TIME * SUBMISSION_INTERVAL) / 2
60 =  (2 * 60) / 2
```